content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -146,68 +177,80 @@ most cases, you should follow the
 [NFT standard](https://developers.metaplex.com/token-metadata/token-standard#the-non-fungible-standard)
 to ensure compatibility with wallets, programs, and applications.
 
-To create the metadata, use the `uploadMetadata` method provided by the SDK.
-This method accepts a metadata object and returns a URI that points to the
-uploaded metadata.
+To create the metadata, use the `uploadJson()` method provided by the SDK. This
+method accepts a metadata object and returns a URI that points to the uploaded
+metadata.
 
 ```typescript
-const { uri } = await metaplex.nfts().uploadMetadata({
-  name: "My NFT",
-  description: "My description",
-  image: imageUri,
+const uri = await umi.uploader.uploadJson({
+  name,
+  description,
+  image,
 });
 ```
 
 #### Create the NFT
 
 After uploading the NFT's metadata, you can finally create the NFT on the
-network. The Metaplex SDK's `create()` method allows you to create a new NFT
-with minimal configuration. This method will create the mint account, token
-account, metadata account, and master edition account for you. The data provided
-to this method will represent the onchain portion of the NFT metadata. You can
-explore the SDK to see all the other input optionally supplied to this method.
+network. The `mplTokenMetadata` plugin we added earlier provides the required
+helpers to create an NFT or any other token with minimal configuration. The
+helper `createNft` method will create the mint account, token account, metadata
+account, and master edition account for you. The data provided to this method
+will represent the onchain portion of the NFT metadata. You can explore the SDK
+to see all the other input optionally supplied to this method.
 
 ```typescript
-const { nft } = await metaplex.nfts().create(
-  {
-    uri: uri,
-    name: "My NFT",
-    sellerFeeBasisPoints: 0,
-  },
-  { commitment: "finalized" },
-);
+const { signature, result } = await createNft(umi, {
+  mint,
+  name: "My NFT",
+  uri,
+  updateAuthority: umi.identity.publicKey,
+  sellerFeeBasisPoints: percentAmount(0),
+}).sendAndConfirm(umi, { send: { commitment: "finalized" } });
```

Comment:
Question - does Umi allow us to set a default confirmation status?

Reply:
for the entire instance I am unsure.

Let me ask in the Metaplex discord.

Reply:
I'm guessing that if we instantiate it with the `Connection` object it inherits the commitment status 

```typescript
// create a new connection to Solana's devnet cluster
const connection = new Connection(clusterApiUrl("devnet"), {
	commitment: "finalized",
});
```

Reply:
The mentioned method wouldn't work -> [Metaplex Discord message link](https://discord.com/channels/848060988636921856/849844496941842432/1277874229928263764)

Solution provided is to create a wrapper function but that's too much to do -> [ Metaplex discord message link](https://discord.com/channels/848060988636921856/849844496941842432/1277965547773886576)

Reply:
Thank Jimmi, I appreciate you chasing this with Metaplex. Understood.